UOP1_Project/Assets/Scripts/Sound/SoundEmitter.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+[RequireComponent (typeof(AudioSource))]
+public class SoundEmitter : MonoBehaviour
+{
+	private AudioSource thisAudioSource;
+
+	private void Awake()
+	{
+		DontDestroyOnLoad(this);
+		thisAudioSource = this.GetComponent<AudioSource>();
+	}
+
+	// Start is called before the first frame update
+	void Start()
+    {
+		thisAudioSource.playOnAwake = false;
+    }
+
+	public void PlaySound(AudioClip clip, AudioMixerGroup mixer, Vector3 position, bool loop, float volume, float pitch = 1)
+	{
+		if (position == Vector3.zero)
+		{
+			thisAudioSource.spatialBlend = 0;
+		}
+		else
+		{
+			thisAudioSource.spatialBlend = 1;
+			this.transform.position = position;
+		}
```

Comment:
Thanks for the input, you are right that this was unnecessary and removes current functionality. If I make this change I guess it would be better to just use one function to play all sounds in sound manager, and avoid having PlaySpatialSound and Play2DSound right? or I conserve those and default the value of spartialBlend to 1 and 0 respectively